Intro_markdown.Rmd

```diff
@@ -232,31 +270,17 @@ There are other commands and various packages that can be used to import dataset
 
 ### Importing data from the Analytical Platform cloud storage (Amazon S3)
 
-Data that has been approved for storage on the Analytical Platform is generally stored in a data source (referred to as a 'bucket') on Amazon S3, which is the cloud storage solution used by the Analytical Platform. There are two options for packages that can be used to import data from S3: `Rs3tools` and `botor`.
-
-#### Importing data using the `Rs3tools` package {-}
-
-The `Rs3tools` package is maintained by other analysts in MoJ and has the advantage of being R-native, resulting in it being quicker to install than `botor` (which requires a Python environment). Here's how it can be installed:
-
-```{r, results="hide"}
-install.packages("moj-analytical-services/Rs3tools")
-```
-
-And here's how to use `Rs3tools` to read in the `offenders` dataset that we'll be using in this session from the 'alpha-r-training' S3 bucket:
-
-```{r, results="hide"}
-offenders <- Rs3tools::s3_path_to_full_df("alpha-r-training/intro-r-training/Offenders_Chicago_Police_Dept_Main.csv")
-```
-
-#### Importing data using the `botor` package {-}
+Data that has been approved for storage on the Analytical Platform is generally stored in a data source (referred to as a 'bucket') on Amazon S3, which is the cloud storage solution used by the Analytical Platform. To import data from S3 we use `botor`.
```

Comment:
See above comment that we should be using Rs3tools

Reply:
have reinstated